src/main/java/vic/cinema/Main.java

```diff
@@ -3,18 +3,21 @@
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.util.List;
+import javax.security.sasl.AuthenticationException;
```

Comment:
I think you need to create your own custom exception

Reply:
Done